src/handle-data.ts

```diff
@@ -113,6 +113,10 @@ export function getColumnInfo(layout: TableLayout, colIdx: number, pageColIdx: n
     ({ qDimensionType } = dimInfo);
   }
 
+  // TODO:
+  // check about hidden columns and why we need to get it from properties
+  const isActivelySorted = colIdx === qEffectiveInterColumnSortOrder[0];
```

Comment:
I can't find any usage of this prop

Reply:
this is being passed to shared lib as field/col/header prop: https://github.com/search?q=repo%3Aqlik-oss%2Fnebula-table-utils%20isActivelySorted&type=code

and it's being used to enable both sort asc and desc button on actively sorted column

Reply:
Just as a side note: 
fixed the Todo thing in this commit: https://github.com/qlik-oss/sn-table/pull/1084/commits/9cbd5c0d57c3d3fd5cd00ca3cea5cbe76e1d9ddd

Reply:
I saw that, nice